Modules/_multiprocessing/semaphore.c

```diff
@@ -435,7 +435,7 @@ semlock_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
     int kind, maxvalue, value, unlink;
     PyObject *result;
     char *name, *name_copy = NULL;
-    static char *kwlist[] = {"kind", "value", "maxvalue", "name", "unlink",
+    static const char *kwlist[] = {"kind", "value", "maxvalue", "name", "unlink",
                              NULL};
```

Comment:
```suggestion
                                   NULL};
```

I think this is the right number of spaces to re-align this...